VimeoUpload/Upload/Model/VideoSettings.swift

```diff
@@ -55,8 +55,12 @@ class VideoSettings: NSObject
         self.privacy = privacy
         self.users = users
         self.password = password
-        self.title = self.trim(title)
-        self.desc = self.trim(description)
+        
+        // Wrap in closure so didSet gets called [CL] 3/25/16
+        ({
```

Comment:
Hey @cjlarsen I saw your comment on the other PR and didn't remember that the setter wouldn't be called in init. I prefer your original approach to the unnamed closure. I think you should revert to just calling `self.trim` in init.